tests/test_indexer.py

```diff
@@ -101,9 +102,13 @@ def indexer_setup(cls, replica):
         Config.set_config(BucketConfig.TEST)
         _, _, cls.test_bucket = Config.get_cloud_specific_handles(cls.replica)
         cls.dss_alias_name = dss.Config.get_es_alias_name(dss.ESIndexType.docs, dss.Replica[cls.replica])
+        create_elasticsearch_index(f"initial_{IndexSuffix.name}", dss.Replica[cls.replica], logger)
```

Comment:
This line (105) should not be necessary, and all the tests pass for me without it. Please remove this line.